doc/release-notes/5.1.1-release-notes.md

```diff
@@ -27,6 +27,10 @@ Larger installations may want to increase the number of open S3 connections allo
 `./asadmin create-jvm-options "-Ddataverse.files.<id>.connection-pool-size=4096"`
 (where `<id>` is the identifier of your S3 file store (likely `"s3"`). The JVM Options section of the [Configuration Guide](http://guides.dataverse.org/en/5.1.1/installation/config/) has more information.
 
+### New S3 Bucket CORS setting for Direct Upload/Download
+
+When using S3 storage with direct upload and/or download enabled, one must now expose the ETag header as documented in the [updated cors.json example](https://guides.dataverse.org/en/latest/developers/big-data-support.html?highlight=etag#s3-direct-upload-and-download).
```

Comment:
```suggestion
When using S3 storage with direct upload and/or download enabled, one must now expose the ETag header as documented in the [updated cors.json example](https://guides.dataverse.org/en/5.1.1/developers/big-data-support.html?highlight=etag#s3-direct-upload-and-download).
```

I prefer linking to version numbers of guides (in this case, 5.1.1) instead of "latest" because we might reorganize pages. It's likely that this big data support content will someday move from the dev guide to the admin guide, for example.